website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+Open source communities are built on the foundation of collaboration, inclusivity, and diversity. Communities comprise a group of individuals with different backgrounds, experiences, and perspectives, who come together to share their knowledge, skills, and passion for creating something meaningful and impactful. To join an open source community, it's important to understand what the community is and what it expects from its members.
+
+The community is not just a group of people working on a project, but rather a living and breathing entity that expects respectful and courteous behavior from its members. It's crucial to be aware of cultural differences and to embrace the genuine spirit of community-based consideration for others. This means being patient, empathetic, and helpful, and striving to create an environment where everyone feels valued and supported.
+
+At the same time, it's important to maintain a level of professional conduct to ensure that contributors don't inadvertently self-sabotage. This includes following the project's code of conduct, adhering to coding standards, and contributing code that is well-tested and documented. ie. The [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
+It's also important to be open to feedback and criticism, and to work collaboratively with other members of the community to improve the project. People of all skill levels work hand-in-hand to improve their projects.
+
+### Choose a Project
+When it comes to contributing to open source, choosing the right project is crucial. A good place to start is the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/), which hosts dozens of popular open source projects in the [cloud native ecosystem](https://landscape.cncf.io/). There's a CNCF project for almost every area of technology, from container orchestration to service mesh to storage. For instance, if you're interested in containerization, you might consider contributing to [Kubernetes](https://kubernetes.io/), the de-facto standard for container orchestration. Or, if you're interested in observability and monitoring, you might consider contributing to [Prometheus](https://prometheus.io/), which provides powerful metrics collection and analysis. Other popular CNCF projects include [Envoy](https://www.envoyproxy.io/), [Fluentd](https://www.fluentd.org/), and [Jaeger](https://www.jaegertracing.io/).
+
+When choosing a project to contribute to, it's important to consider your interests and skill level. Look for projects that align with your strengths and that you're passionate about. CNCF projects have a large and supportive community, with many resources available for contributors. It is very important to start by reading the project documentation and exploring the issue tracker. If you're new to the project, look for issues labeled as "good first issue" or "help wanted" - these are often small, well-defined tasks that are a good place to start.
```

Comment:
Do we know if clotributor ready for us to link to when we mention "help wanted" and "good first issues"? 

https://clotributor.dev

It's a site that highlights those issues across CNCF projects.

Reply:
Removed CLOTributor. Can add it later when it is further developed